example-nextjs/pages/index.tsx

```diff
@@ -56,6 +57,23 @@ eratosthenes(100)`)
       .then((pyodide) => setPyodide(pyodide))
   }, [])
 
+  useEffect(() => {
+    if (pyodide) {
+      window.addEventListener('keypress', (e) =>
```

Comment:
I think we can be a bit more efficient here. This will run on ANY key press through the page, but we can probably isolate it to specific key presses within the editor field only.